src/server.tsx

```diff
@@ -58,6 +58,7 @@ app.get('/healthcheck', (req: express.Request, res: express.Response) => {
 interface Epic {
     html: string;
     css: string;
+    js: Function | string;
```

Comment:
If we allow this to be Function, we create a contract between component and platform, begging questions like how does the platform know what to pass the function?

There may be cases in the future where we need to use platform info within components, but generally passing the info from the platform in the metadata of the request seems easier.

For example, if we really wanted the platform to specify stage, doing it in the server request would be a lot easier. (But even that I wouldn't be keen on for the reasons we've already discussed.)

Reply:
My bad, that was never supposed to be `Function | string` I was just trying different things and forgot to revert that to `string`. We no longer need the platform to pass a stage.